packages/v4/patternfly-docs/content/design-guidelines/components/button/button.md

```diff
@@ -5,7 +5,44 @@ section: components
 
 A **button** is a box area or text that communicates and triggers user actions when clicked or selected. Buttons can be used to communicate and immediately trigger actions a user can take in an application, like submitting a form, canceling a process, or creating a new object. Buttons can also be used to take a user to a new location, like another page inside of a web application, or an external site such as help or documentation.
 
-## Button types
+## Usage
+There are certain cases where specific buttons must be used within your UI.
+
+Follow these guidelines for buttons in wizards (shown in the image below):  
+* The **Next** button should always be a [primary button](#primary-button).
+* The **Back** button should always be a [secondary button](#secondary-button).
+* The **Cancel** button should always be a [link button](#link-button).
+
+<img src="./img/wizard.png" alt="Example of buttons in a wizard"/>
+
+### When to use 
```

Comment:
Now I'm like, this should probably be moved to the link button section since it's just a series of examples of when to use a link button specifically ?